include/functions.h

```diff
@@ -1258,8 +1258,8 @@ void* Graph_Alloc2(GraphicsContext* gfxCtx, size_t size);
 void Graph_OpenDisps(Gfx** dispRefs, GraphicsContext* gfxCtx, const char* file, int line);
 void Graph_CloseDisps(Gfx** dispRefs, GraphicsContext* gfxCtx, const char* file, int line);
 #endif
-Gfx* Gfx_Open(Gfx* gfx);
-Gfx* Gfx_Close(Gfx* gfx, Gfx* dst);
+Gfx* Gfx_Open(Gfx* gfxDisp);
+Gfx* Gfx_Close(Gfx* gfxDisp, Gfx* gfx);
```

Comment:
why "disp" ? afaik that stands for "display"

Reply:
opted for disp to make the distinction that the first argument should be a display buffer pointer (i.e. POLY_OPA_DISP) rather than any old Gfx*.